service/ServiceClient.py

```diff
@@ -0,0 +1,72 @@
+import string
+
+from domain.Client import Client
+
+class ServiceClient(object):
+
+    def __init__(self, __repo):
+        self.__repo = __repo
+
+    def add(self,id,name,description ):
+        '''
+        :param id: int
+        :param name: string
+        :param description: string
+        :return:
+        raises value error if id already exists
+        '''
+        if not(isinstance ( id, int)):
```

Comment:
Validarile de tipul asta se vor face in controller ( resti API ). Vom da bad request ( 404 ) cand primim un model care nu respecta cerintele.